### PR TITLE
configure maven central release job

### PR DESCRIPTION
**Pull request by mpollmeier**

### CODE REVIEW

**Comment by mpollmeier** [comment]
that was a bit over-eager, sorry

**Comment by mpollmeier** [comment]
better now - ready for review

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        run: sudo apt update && sudo apt install -y gnupg
+      - name: Import GPG key
+        run: echo $PGP_SECRET | base64 --decode | gpg --batch --import
+        env:
+          PGP_SECRET: ${{ secrets.PGP_SECRET }}
+      - name: Set up JDK
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: 19
```
Comment (DavidBakerEffendi) [inline .github/workflows/release.yml:26]: Just a general thing to note across our CI, releases for Java 19 have stopped for a while and perhaps we should look at LTE, either 17 or 21: https://en.wikipedia.org/wiki/Java_version_history

**Review by DavidBakerEffendi — APPROVED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (mpollmeier) [reply .github/workflows/release.yml]: good point, changed to 21

**Comment by mpollmeier** [comment]
let's see what happens 🦀

**Pull request merged**